.circleci/config.yml

```
@@ -5,13 +5,13 @@ orbs:
 jobs:
   update-helm-charts-index:
     docker:
-      - image: docker.mirror.hashicorp.services/circleci/golang:1.19.2
+      - image: docker.mirror.hashicorp.services/golang:1.19.2
```

Ah, yeah I think the circle images are now under `cimg`. So this might work too:

```suggestion
      - image: docker.mirror.hashicorp.services/cimg/go:1.19.2
```

https://circleci.com/blog/announcing-our-next-generation-convenience-images-smaller-faster-more-deterministic/

---

Oh awesome, TIL, thanks!